tests/test_scripts.py

```diff
@@ -446,3 +446,43 @@ def test_combine_outfile(tmpdir, script_runner):
     combined_ws = pyhf.Workspace(combined_spec)
     assert combined_ws.channels == ['channel1', 'channel2']
     assert len(combined_ws.measurement_names) == 8
+
+
+@pytest.mark.parametrize('do_json', [False, True])
+@pytest.mark.parametrize(
+    'algorithms', [['md5'], ['sha256'], ['sha256', 'md5'], ['sha256', 'md5']]
+)
+def test_workspace_digest(tmpdir, script_runner, algorithms, do_json):
+    results = {
+        'md5': '202eb7615102c35ba86be47eb6fa5e78',
+        'sha256': '7c32ca3b8db75cbafcf5cd7ed4672fa2b1fa69e391c9b89068dd947a521866ec',
+    }
+
+    temp = tmpdir.join("parsed_output.json")
+    command = 'pyhf xml2json validation/xmlimport_input/config/example.xml --basedir validation/xmlimport_input/ --output-file {0:s} --hide-progress'.format(
+        temp.strpath
+    )
```

Comment:
You had gotten in the habit of using f-strings. Can we keep this going here? I like the adoption! :)